Doc/tools/touch-clean-files.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+"""
+Touch files that must pass Sphinx nit-picky mode
+so they are rebuilt and we can catch regressions.
+"""
+
+from pathlib import Path
+
+# Input file has blank line between entries to reduce merge conflicts
+CLEAN = Path("Doc/tools/clean-files.txt").read_text(encoding="UTF-8").split("\n")
+
+print("Touching:")
+for filename in CLEAN:
+    if filename:
+        print(filename)
+        Path(filename).touch()
```

Comment:
```suggestion
    if filename.strip():
        print(filename)
        Path(filename.strip()).touch()
```

Avoid any issues with trailing (or leading) whitespace, either on empty or non-empty lines.

Reply:
This could be walrused to avoid duplication, or if my previous suggestion is accepted it could be added to `CLEAN = [fname.strip() ...]` directly and both the `if` and the `strip` removed.

Another option is to avoid the `strip` and let it fail if there is extra whitespace so that the user can remove it.  If we do this, we should provide an easy way for the user to test this locally before creating the PR, either as part of `make patchcheck` (is it still used?), or as a target in the `Doc`'s `makefile`.  This can be done in a separate PR though.

Reply:
> if my previous suggestion is accepted it could be added to `CLEAN = [fname.strip() ...]` directly and both the `if` and the `strip` removed.

+1 to that approach

> Another option is to avoid the `strip` and let it fail if there is extra whitespace so that the user can remove it.

As much as I'm pretty nitpicky, this seems like being a bit too much of a stickler for even my taste to fail the whole CIs for considering its trivial to ignore :)